azure-pipelines.yml

```diff
@@ -1,6 +1,6 @@
 ---
 trigger:
-  - master
+  - none
```

Comment:
why are you changing trigger for the main pipeline?

Reply:
changed to master